pkg/apis/serving/v1alpha1/kfservice_types.go

```diff
@@ -78,11 +78,10 @@ type CustomSpec struct {
 
 // KFServiceStatus defines the observed state of KFService
 type KFServiceStatus struct {
-	Conditions                ConditionsSpec  `json:"conditions,omitempty"`
-	URI                       URISpec         `json:"uri,omitempty"`
-	Revisions                 []RevisionsSpec `json:"revisions,omitempty"`
-	LatestCreatedRevisionName string          `json:"latestCreatedRevisionName,omitempty"`
-	LatestReadyRevisionName   string          `json:"latestReadyRevisionName,omitempty"`
+	Conditions StatusConditionsSpec    `json:"conditions,omitempty"`
+	URI        URISpec                 `json:"uri,omitempty"`
+	Default    StatusConfigurationSpec `json:"default,omitempty"`
```

Comment:
Please yay or nay "default"

Otherwise I think the next two options were "stable" or "primary".